classes/CMSCategory.php

```diff
@@ -496,28 +577,34 @@ public function getLink(Link $link = null)
         return $link->getCMSCategoryLink($this->id, $this->link_rewrite);
     }
 
-    public function getName($id_lang = null)
+    /**
+     * @param null $idLang
+     *
+     * @return string
+     */
+    public function getName($idLang = null)
     {
         $context = Context::getContext();
-        if (!$id_lang) {
+        if (!$idLang) {
```

Comment:
Hmmz, I wanted to recommend `empty`, but now that we have 'contracts' in the PHPDoc comments, it's much better to go for your suggestion ;)